packages/library/playwright.config.ts

```diff
@@ -17,6 +17,8 @@ export default defineConfig({
   use: {
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: "on-first-retry",
+    viewport: { width: 1280, height: 800 }, // Set default viewport
+    headless: true,
```

Comment:
Why `headless: true` ? can we remove this config? For me headless was the default. Running tests with --ui brought up the interface.